### PR TITLE
Add content/es/docs/reference/glossary/sysctl.md

### PR DESCRIPTION
**Pull request by jluis**
refs #13948

### CODE REVIEW

File: content/es/docs/reference/glossary/sysctl.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+date: 2019-02-12
+full_link: /docs/tasks/administer-cluster/sysctl-cluster/
+short_description: >
+  Una interfaz para obtener y asignar valores a parámetros del núcleo Unix activo.
+
+aka:
+tags:
+- tool
+---
+ `sysctl` es una interfaz común usadapara consultar o modificar artributos del
```
Comment (alexbrand) [inline content/es/docs/reference/glossary/sysctl.md]: s/usadapara/usada para

File: content/es/docs/reference/glossary/sysctl.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+aka:
+tags:
+- tool
+---
+ `sysctl` es una interfaz común usadapara consultar o modificar artributos del 
+ núcleo Unix durante su ejecución.
+
+<!--more-->
+
+En los sistemas Unix-like, `sysctl` es el comando que usan los administradores,
```
Comment (sftim) [inline content/es/docs/reference/glossary/sysctl.md]: “Unix-like”?

File: content/es/docs/reference/glossary/sysctl.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+para ver o modificar esos valores y también el nombre de la llamada al sistema
+que realiza esta función.
+
+{{< glossary_tooltip text="Container" term_id="container" >}} runtimes y
```
Comment (sftim) [inline content/es/docs/reference/glossary/sysctl.md]:
```suggestion
{{< glossary_tooltip text="Contenedor" term_id="container" >}} runtimes y
```
maybe?
Also, “runtimes” is English and with no expansion into Spanish. Is that OK?

File: content/es/docs/reference/glossary/sysctl.md (hunk 2 above)
Comment (jluis) [reply content/es/docs/reference/glossary/sysctl.md]: I was socked when the Spanish did't translate the term so I think if it is good enough for the Wikipedia https://es.wikipedia.org/wiki/Unix-like it  shall be for us

File: content/es/docs/reference/glossary/sysctl.md (hunk 3 above)
Comment (jluis) [reply content/es/docs/reference/glossary/sysctl.md]: I'm OK with the proposed change I only used "Container" after grepping the es/ dir and found  on  content/es/docs/concepts/architecture/nodes.md that used container runtime.
In technical Spanish literature is used untranslated, I've only seen translated on books published before 1990 and the translations would sound like a bad joke.

Comment (jluis) [reply content/es/docs/reference/glossary/sysctl.md]: Rethinking the paragraph, and with the help form the Wikipedia to eliminate any personal bias, I think that the new translation reflects better the intention of the English sentence:

"La ejecución del contenedor y de los complementos de red puede depender de los valores assignados via sysctl."

File: content/es/docs/reference/glossary/sysctl.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+date: 2019-02-12
+full_link: /docs/tasks/administer-cluster/sysctl-cluster/
+short_description: >
+  Una interfaz para obtener y asignar valores a parámetros del núcleo Unix activo.
+
+aka:
+tags:
+- tool
+---
+ `sysctl` es una interfaz común usada para consultar o modificar artributos del
```
Comment (electrocucaracha) [inline content/es/docs/reference/glossary/sysctl.md]:
```suggestion
 `sysctl` es una interfaz común usada para consultar o modificar atributos del 
```

File: content/es/docs/reference/glossary/sysctl.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+ núcleo Unix durante su ejecución.
+
+<!--more-->
+
+En los sistemas Unix-like, `sysctl` es el comando que usan los administradores,
+para ver o modificar esos valores y también el nombre de la llamada al sistema
+que realiza esta función.
+
+La ejecución del {{< glossary_tooltip text="Contenedor" term_id="container" >}} 
+y de los complementos de red puede depender de los valores assignados via `sysctl`.
```
Comment (electrocucaracha) [inline content/es/docs/reference/glossary/sysctl.md]:
```suggestion
y de los complementos de red puede depender de los valores asignados via `sysctl`.
```

**Comment by raelga** [comment]
Thanks @jluis! 💙 

`lgtm` after fixing the changes requested by @electrocucaracha.

**Comment by raelga** [comment]
Tracked at #13948.

**Comment by raelga** [comment]
Gracias @jluis!

Podrías hacer un squash de los commits y dejarlo en uno solo? :octocat:

**Comment by jluis** [comment]
fet

**Comment by raelga** [comment]
Muchas gracias!

/lgtm

**Comment by raelga** [comment]
Muchas gracias @jluis, @alexbrand y @electrocucaracha !

💙

**Pull request merged**